install.js

```diff
@@ -108,12 +117,12 @@ function findSuitableTempDirectory(npmConf) {
 }
 
 
-function getRequestOptions(proxyUrl) {
+function getRequestOptions(proxyUrl, downloadPath) {
   var options
   if (proxyUrl) {
-    options = url.parse(proxyUrl)
-    options.path = downloadUrl
-    options.headers = { Host: url.parse(downloadUrl).host }
+    options = url.parse(proxyUrl) 
+    options.path = dowloadPath
```

Comment:
There is a typo here on `dowloadPath`, should be `downloadPath`. It seems we are not testing install with a proxy... Anyway, please fix it.